.profile.d/01-remove-installers.sh

```diff
@@ -0,0 +1,5 @@
+rm -rf $HOME/web/wp/wp-admin/install.php
```

Comment:
This is not going to work in wp-bedrock-seed projects.

$HOME is /app in heroku but the actual filepath would be ./bedrock-on-heroku/web/wp/wp-admin/install.php

Since this is a bash script anyway, we could check if the directory "bedrock-on-heroku" exists, in order to know the right location to remove those files.